tests/test_resultinfo.py

```diff
@@ -72,6 +71,10 @@ def test_print_result_info(model):
     print(model.metadata.result_info)
 
 
+def test_result_info_available_qualifier_labels(model):
+    assert model.metadata.result_info.available_qualifier_labels == []
+
+
```

Comment:
```suggestion
```
Remove test as it blocks the fix due to the CI still being on 232, and ansys-dpf-gate not being updated in dpf-standalone 232